### PR TITLE
complete errors-debug-profiling

### PR DESCRIPTION
**Pull request by CuongNgMan**

### CODE REVIEW

**Review by viktordavidovich — COMMENTED** [review]
You can use this article to implement a better way of creating custom error types in JS 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error#custom_error_types

File: errors-debug-profiling/exercise-2.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# onboarding.epam.com
+## Total loading time: 4.16s
+## Took 3.92s to load DOM contents
+## Below HTTP request took 657ms to response
+
+```
+https://menu.epam.com/scripts/implementation.js?app=onboarding
+```
+
+## No mem-leak
```
Comment (viktordavidovich) [inline errors-debug-profiling/exercise-2.md]: I believe we should measure other metrics. Like it's being said in the task 

• Do they have memory leaks? 
• What functions are taking more time for execution? 
• What time does it take for rendering first meaningful page? 

1. Let's discuss during regular call the way you were checking page for "Memory leaks"
2. Let's make sure we find JS functions which are taking more time for execution, but not Network events.  
3. I believe we need to check the following metric to answer the question https://web.dev/lcp/

Comment (CuongNgMan) [reply errors-debug-profiling/exercise-2.md]: Thank you for the helpful resources. Please take a look at an improvement from 20f914d commit

**Review by viktordavidovich — APPROVED** [review]
Great job!

**Review by viktordavidovich — APPROVED** [review]

**Pull request merged**